cysync/src/pages/initialApp/deviceFlow/deviceSetup/stepperFormComponents/upgradingDevice.tsx

```diff
@@ -220,6 +222,7 @@ const UpgradingDevice: React.FC<StepComponentProps> = ({
               <CustomButton
                 onClick={refreshComponent}
                 style={{ marginTop: '2rem' }}
+                disabled={inBackgroundProcess || !deviceConnection}
```

Comment:
Should not be disabled when the button says `Ok`

Reply:
Got it